### PR TITLE
Fix types expander in root for 5.2 (for non-DX Plone Site Root)

### PR DESCRIPTION
**Pull request by sneridagh**
Related to: https://github.com/plone/volto/pull/4946

The tests failed for 5.2 when trying to use the expander in a non-DX Plone site root.

I updated also the build for 5.2 to latest Plone 5.

### CODE REVIEW

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by davisagli — APPROVED** [review]
@sneridagh Thanks, LGTM

**Comment by sneridagh** [comment]
@davisagli whenever you have time, a release with this one and the images serializers would be great! Thanks!

File: news/1669.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix types expander in root for 5.2 (for non-DX Plone Site Root) @sneridagh
```
Comment (davisagli) [inline news/1669.bugfix:1]:
```suggestion
Fix types expander in root for Plone 5.2 (for non-Dexterity Plone Site Root) @sneridagh
```

**Pull request merged**